src/js/player.js

```diff
@@ -290,22 +290,23 @@ class Player extends Component {
 
     // If language is not set, get the closest lang attribute
     if (!options.language) {
-      if (typeof tag.closest === 'function') {
-        const closest = tag.closest('[lang]');
```

Comment:
I think here particularly, we should avoid calling `closest` twice, given that it's a DOM operation.

Reply:
This is a hard one, no early return allowed, but about DOM operation, I think it only slow when involve with render html (this case is only compare).

I'll close this, been thinking but there is no good solution in this case, even without performance, 2 if is bit gross to write.